conf_files/testing.yaml

```diff
@@ -85,6 +85,8 @@ focusing:
     frequency_unit: hour
     timeout: 600
     filter_name: one
+    dummy_ra: 300
```

Comment:
It might be better to specify this as an `alt-az` for now. I think you can drop the "dummy" in the variable name as its not really a dummy. So just `alt` and `az` are fine. You could even just make it use the flat field alt / az that is already in the config, but up to you.